<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingDialog</class>
 <widget class="QWidget" name="SettingDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>725</width>
    <height>555</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>设置</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="styleSheet">
      <string notr="true">font: 12px &quot;华文楷体&quot;;</string>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="paramTab">
      <attribute name="icon">
       <iconset resource="MainDlg.qrc">
        <normaloff>:/MainDlg/Resources/images/statistics.ico</normaloff>:/MainDlg/Resources/images/statistics.ico</iconset>
      </attribute>
      <attribute name="title">
       <string>参数配置</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="utility::QTreeWidgetEx" name="treeWidget">
         <column>
          <property name="text">
           <string notr="true">1</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="channelTab">
      <attribute name="icon">
       <iconset resource="MainDlg.qrc">
        <normaloff>:/MainDlg/Resources/images/statistics.ico</normaloff>:/MainDlg/Resources/images/statistics.ico</iconset>
      </attribute>
      <attribute name="title">
       <string>通道配置</string>
      </attribute>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="saveButton">
       <property name="text">
        <string>保存</string>
       </property>
       <property name="icon">
        <iconset resource="MainDlg.qrc">
         <normaloff>:/MainDlg/Resources/images/new_save.ico</normaloff>:/MainDlg/Resources/images/new_save.ico</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="exitButton">
       <property name="text">
        <string>退出</string>
       </property>
       <property name="icon">
        <iconset resource="MainDlg.qrc">
         <normaloff>:/MainDlg/Resources/images/quit.ico</normaloff>:/MainDlg/Resources/images/quit.ico</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>utility::QTreeWidgetEx</class>
   <extends>QTreeWidget</extends>
   <header location="global">Utility/Control/QTreeWidgetex.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="MainDlg.qrc"/>
 </resources>
 <connections/>
</ui>
